### ISSUE TITLE
Voice Over iOS users cannot select an option of the ComboBox component

### ISSUE THREAD

**Issue by dylankil** [OfficeDev/office-ui-fabric-react#8930]
## Codepen demonstrating the issue
https://codepen.io/dylankil/pen/XQvVwj?&editable=true 

## Actual behavior:
Voice Over focus stays on the ComboBox input

## Expected behavior:
Voice Over focus moves to currently selected option in the Callout

**Comment by atneik** [comment]
Hi @dylankil, I tried to use VoiceOver on my phone and was able to select an option using the given codepen. Here is a screen capture:
[KPWW5121_Trim.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3140330/KPWW5121_Trim.zip)

Should I be expecting something else? If there is a sequence that doesn't work please let me know.

**Comment by dylankil** [comment]
Apologies. The ComboBox needs to be inside another Surface UI Component, such as Panel. I've updated the codepen.

**Comment by atneik** [comment]
Ahh now I see the issue. Thanks!

**Comment by dylankil** [comment]
Yup, in the mean-time I'm thinking a work-around may be to append an aria-hidden="true" to other divs that have 'ms-Layer' in their classList.

**Comment by atneik** [comment]
@dylankil I don't understand how the work-around will help. Wouldn't it just make the whole layer not accessible when VoiceOver is on?

**Comment by dylankil** [comment]
Would be something like this in HTML:

<div id="reactApp" aria-hidden="true">…</div>
<div class="ms-Layer ms-Layer--fixed root-87" data-portal-element="true" aria-hidden="true">…</div>
<div class="ms-Layer ms-Layer--fixed root-87" data-portal-element="true">…</div>

Where the bottom two elements are not accessible when the top-most surface UI is visible.

**Comment by atneik** [comment]
That makes sense, Surfaces that block the background UI could set  `aria-hidden` as `true` on lower layers. Would you have bandwidth to contribute this fix? Or else I'll mark this bug for our backlog.

**Comment by atneik** [comment]
@dylankil how about using [`aria-modal` ](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html) instead of above mentioned `aria-hidden` approach?

**Comment by atneik** [comment]
I noticed that Surfaces already uses `aria-modal` method and it still doesn't work for iOS. Looks like we will have to implement above mentioned `aria-hidden` method for all Surfaces. Putting it in our backlog, unless @dylankil you'll have bandwidth to contribute?

**Comment by dylankil** [comment]
@atneik Tried the aria-hidden work-around within the scope of the Immersive Reader application React business logic and it didn't work. Voice Over fails to focus on the Callout element.

**Comment by jspurlin** [comment]
@dzearing This is a voiceOver bug where they are not responding to aria-activedecendent correctly (notice how narrator knows the combobox item that is being "focused").

See https://www.w3.org/TR/wai-aria-1.1/#combobox where it describes how aria-activedecendent should be used (whic his what we are doing):

>  ...in lieu of moving focus, such keyboard mechanisms can control the value of aria-activedescendant on the textbox element. When a descendant of the popup element is active, authors MAY set aria-activedescendant on the textbox to a value that refers to the active element within the popup while focus remains on the textbox element.
